directory/directory_dest.go

```diff
@@ -172,7 +173,7 @@ func (d *dirImageDestination) PutBlob(ctx context.Context, stream io.Reader, inp
 		return types.BlobInfo{}, err
 	}
 	succeeded = true
-	return types.BlobInfo{Digest: computedDigest, Size: size}, nil
+	return types.BlobInfo{Digest: computedDigest, Size: size, MediaType: manifest.DockerV2SchemeLayerMediaTypeUncompressed}, nil
```

Comment:
Why is this the right value?
- `DesiredLayerCompression` is `PreserveOriginal`
- OCI has different MIME types
- configs have different MIME types

Also, it’s not at all obvious to me that `PutBlob` has any business deciding what the content of the input is; guessing is unreliable, and `PutBlob` has no knowledge of = can’t support `!imageCopier.canModifyManifest`.

Reply:
I guess we need to leave this decision to copy then :) Thanks!